tests/TestFingerprintMatcher.py

```
@@ -0,0 +1,198 @@
+            self.assertGreaterEqual(candidates[0].confidence, ACCEPT_THRESHOLD)
+
+
+class TestActivationStructLayout(unittest.TestCase):
+    """ctypes layout of the WASAPI activation params (needs comtypes)."""
+
+    def test_struct_sizes(self):
+        try:
+            import ctypes
+
+            from src.sound_trigger.capture import process_loopback as pl
+        except Exception as exc:  # comtypes/psutil not installed in this env
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)